sdk/servicebus/service-bus/CHANGELOG.md

```diff
@@ -1,13 +1,15 @@
 # Release History
 
-## 1.1.6 (TBD)
+## 1.1.6 (2020-04-23)
 
 - Removes the `@azure/ms-rest-nodeauth` dependency.
   This allows users to use any version of `@azure/ms-rest-nodeauth` directly with `@azure/service-bus` without TypeScript compilation errors.
   Fixes [bug 8041](https://github.com/Azure/azure-sdk-for-js/issues/8041).
 - Fixes an issue where non-retryable errors caused by a connection disconnecting were not getting surfaced to the user's registered error handler
   when using the `registerMessageHandler` method on a receiver.
   [PR 8401](https://github.com/Azure/azure-sdk-for-js/pull/8401)
+- Updates the way connection disconnects are handled to create a new connection object rather than re-using the existing one.
+  This should reduce some issues caused when the existing connection gets into an unexpected state.
```

Comment:
```suggestion
  This should reduce some issues caused when the existing connection gets into an unexpected state.
  [PR 8447](https://github.com/Azure/azure-sdk-for-js/pull/8447)
 - Adds a new method `open()` on the sender to allow you to front load the work of setting up the underlying AMQP links. Use this if you want to avoid having your first `send()` operation pay the tax of link set up.
   [PR 8329](https://github.com/Azure/azure-sdk-for-js/pull/8329). This PR also fixes a bug where a sender recovering from connection loss does not report the error back to the user from ongoing send operations.
```

Reply:
@richardpark-msft, you ok with the text above for your open() feature?

Reply:
"Some issues" has me worried. Are there any more known issues? Otherwise I'd say "Fixes reconnection issues by..."

Reply:
I went with @bterlson's wording.